cli/test/test_index_data.py

```diff
@@ -30,33 +30,22 @@ def test_get_document_generator(test_input_dir: Path):
 
     for doc in doc_generator:
         for field in [
-                    "document_id",
-                    "document_name",
-                    "document_description",
-                    "document_source_url",
-                    "document_cdn_object",
-                    "document_md5_sum",
-                    "translated",
-                    "document_slug",
-                    "document_content_type",
-                ]:
+            "document_id",
+            "document_name",
+            "document_description",
+            "document_source_url",
+            "document_cdn_object",
+            "document_md5_sum",
+            "translated",
+            "document_slug",
+            "document_content_type",
+            "document_geography",
+            "document_category",
+            "document_source",
+            "document_type",
+            "document_date",
+        ]:
```

Comment:
should we also check for "metadata" if we're still leaving it in the top-level object?

Reply:
it's no longer in the top-level object - see above comment